lib/wallet/rpc.js

```diff
@@ -2005,6 +2006,35 @@ class RPC extends RPCBase {
     return {name, force, account};
   }
 
+  async sendOpenIdempotent(args, help) {
+    const opts = this._validateOpenIdempotent(args, help, 'sendopenidempotent');
+    const wallet = this.wallet;
+    const tx = await wallet.sendOpen(opts.name, opts.force, {
+            account: opts.account,
+            idempotencyKey: opts.idempotencyKey
+          });
```

Comment:
Weird indentation here

Reply:
Ahh yeah, the 'format on save' made the change set way bigger bc the codebase doesn't conform to whatever settings we use elsewhere.

Reply:
Yeah hsd uses different .eslint settings, I recall some weirdness setting it up on my machine but don't remember everything involved. I forget if you need to use "bslint" (bcoin's eslint with frozen deps) or not